src/main/resources/application.properties

```diff
@@ -1,15 +1,17 @@
 server.port=8085
-spring.datasource.url=jdbc:h2:mem:test
-spring.datasource.driverClassName=org.h2.Driver
-spring.datasource.username=sa
-spring.datasource.password=password
-spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
-spring.h2.console.enabled = true;
+spring.datasource.url=jdbc:postgresql://localhost:5432/data
+spring.datasource.username=postgres
+spring.datasource.password=user
 spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
?

Reply:
we do not do that. u should use liquibase changelogs

Reply:
I'll add liquibase to project